test/recipes/30-test_evp_data/evpkdf_x942.txt

```diff
@@ -97,3 +97,21 @@ Ctrl.hexacvp-info = hexacvp-info:a020299D468D60BC6A257E0B6523D691A3FC1602453B35F
 Ctrl.hexukm = hexukm:012345
 Output = C2E6A0978C24AF3932F478583ADBFB5F57D491822592EAD3C538875F46EB057A
 Result = KDF_DERIVE_ERROR
+
+Version = <3.4.0
+KDF = X942KDF-ASN1
+Ctrl.digest = digest:SHAKE-128
+Ctrl.hexsecret = hexsecret:6B
+Ctrl.use-keybits = use-keybits:0
+Ctrl.cekalg = cekalg:id-aes128-wrap
+Ctrl.hexacvp-info = hexacvp-info:a020299D468D60BC6A257E0B6523D691A3FC1602453B35F308C762FBBAC6069A88BCa12080D49BFE5BE01C7D56489AB017663C22B8CBB34C3174D1D71F00CB7505AC759Aa2203C21A5EA5988562C007986E0503D039E7231D9F152FE72A231A1FD98C59BCA6Aa320FD47477542989B51E4A0845DFABD6EEAA465F69B3D75349B2520051782C7F3FC
+Result = KDF_DERIVE_ERROR
+
+Version = >=3.4.0
```

Comment:
Same as above

Reply:
Same answer :grin:

Shake is nonsensical for these KDFs.  It makes sense to error out in the default provider as well.